sqlglot/optimizer/simplify.py

```diff
@@ -227,15 +249,17 @@ def simplify_not(expression: exp.Expression, dialect: Dialect) -> exp.Expression
                         exp.not_(condition.left, copy=False),
                         exp.not_(condition.right, copy=False),
                         copy=False,
-                    )
+                    ),
+                    copy=False,
```

Comment:
Seems like we're copying in main as well. Did you compare this branch to main w/ `copy=False` in both?